.github/workflows/build.yml

```diff
@@ -204,9 +204,10 @@ jobs:
         run: gmake VERBOSE=1
       - name: Run unit tests
         run: .build/target/${{env.ARTIFACT}}/${{env.ARTIFACT}}-test utest --verbose --jobs 1
-      - name: Run unit tests with memcheck
-        run: |
-          for test in $(.build/target/${{env.ARTIFACT}}/${{env.ARTIFACT}}-test utest --list --suppress); do \
-            echo "***** MEMCHECK $test *****"; \
-            valgrind ${{env.VALGRIND_ARGS}} .build/target/${{env.ARTIFACT}}/${{env.ARTIFACT}}-test utest --verbose --jobs 1 --nofork --debug $test; \
-          done
\ No newline at end of file
+# Find an alternativ to valgrind
```

Comment:
I think at least here we can build with address sanitizer and run unit tests with address sanitizer but without valgrind. See the `arch_linux_asan` job.